test_unstructured/partition/test_epub.py

```diff
@@ -11,6 +11,7 @@ def test_partition_epub_from_filename():
     elements = partition_epub(filename=filename)
     assert len(elements) > 0
     assert elements[0].text.startswith("The Project Gutenberg eBook of Winter Sports")
+    assert elements[0].metadata.section is not None
```

Comment:
Can we assert what the name of the section should be instead of just check that it's not `None`? One other good test here would be to get `all_sections = set([element.metadata.section for element in elements])` and check that all the sections we're expecting are present.

Reply:
That's what the test was before, but not all elements have a section field--the issue I was running into before was using the toc as the primary means of parsing, but it's not totally reliable in that multiple toc elements point to the same text and some sections of text aren't included in the toc. I defaulted to having those parts that don't have a corresponding toc element to being `None`, but that could be changed to "NA" or something else.

That said, the toc isn't totally reliable it seems--it doesn't fully align with the actual chapters in the book (which is why a chunk of chapter 2 of the test epub book was missing in the elements when I was just using the toc for parsing)

Reply:
Just looking for something along the lines of this (using the actual expected values for the test document)

```python
assert elements[0].metadata.section == "ch1"

all_sections = set([el.metadata.section for el in elements])
assert all_sections = expected_sections
```